thavalon-server/src/lobby/client.rs

```diff
@@ -258,6 +269,7 @@ impl PlayerClient {
                         game_msg,
                         client_id
                     );
+                    let game_msg = OutgoingMessage::GameMessage(game_msg);
```

Comment:
I think it's more Rusty. The variable really isn't changing, it's just in a different form. Rust allows masking to prevent declaring three variables for basically the same thing, like game_msg, outgoing_game_msg, and outgoing_game_msg_json, for example.